### PR TITLE
fix(taskframework): cannot preview latest log for dlm

### PR DESCRIPTION
**Pull request by krihy**
#### What type of PR is this?

type-bug
module-taskframework
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by smallsheeeep** [comment]
This PR seems do not solve the problem that cannot download full log files when enable task framework.

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/logger/LogUtils.java [hunk 1]
```diff
@@ -44,34 +44,34 @@ public class LogUtils {
 
     public static String getLogContent(String file, Long fetchMaxLine, Long fetchMaxByteSize) {
```
[the remark this answers is not in the document]
Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/logger/LogUtils.java:45]: ok, getLogContent -> getLogContentReversed

**Comment by krihy** [comment]
> This PR seems do not solve the problem that cannot download full log files when enable task framework.

yes, current download dlm  task log is just download preview log,  this version is not urgent and has  discussed with @guowl3, we can solve later version

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/logger/LogUtils.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (smallsheeeep) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/logger/LogUtils.java]: 😂 Although the name doesn't matter, it's better to use `getLatestLogContent`

**Comment by smallsheeeep** [comment]
> > This PR seems do not solve the problem that cannot download full log files when enable task framework.
> 
> yes, current download dlm task log is just download preview log, this version is not urgent and has discussed with @guowl3, we can solve later version

okay

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/logger/LogUtils.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (krihy) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/logger/LogUtils.java]: DONE

**Review by smallsheeeep — APPROVED** [review]
LGTM

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**